perma_web/api/resources.py

```diff
@@ -451,17 +453,35 @@ def obj_create(self, bundle, **kwargs):
                 content_type='image/png',
             ).save()
 
-            # kick off capture task
-            run_task(proxy_capture.s(link.guid, bundle.request.META.get('HTTP_USER_AGENT', '')))
+            try:
+                run_task(proxy_capture.s(link.guid, bundle.request.META.get('HTTP_USER_AGENT', '')))
+            except Exception as e:
+                print "Getting error:",e
```

Comment:
Is this just left over from debugging, or is there something we want to do around exception handling here?